bin/blanksky

```diff
@@ -375,53 +401,55 @@ def apply_gain(infile,outfile,kw,tmpdir,verbose):
         # some columns required by acis_process_events.  The error introduced
         # by this mismatched calibration should not be very significant for
         # the faint, extended objects for which people use these backgrounds.
-        acis_process_events.punlearn()
-
-        acis_process_events.outfile = outfile
-        acis_process_events.acaofffile = "NONE"
-        acis_process_events.stop = "none"
-        acis_process_events.doevtgrade = "no"
-        acis_process_events.apply_cti = "yes"
-        acis_process_events.apply_tgain = "no"
-        acis_process_events.calculate_pi = "yes"
-        acis_process_events.pix_adj = "NONE"
-        acis_process_events.gainfile = f"{os.environ['CALDB']}/data/chandra/acis/det_gain/{gainfile}"
-        acis_process_events.eventdef = eventdef
-        acis_process_events.clobber = "yes"
-        acis_process_events.verbose = verbose
+
+        a_p_e.punlearn()
+
+        a_p_e.outfile = outfile
+        a_p_e.acaofffile = "NONE"
+        a_p_e.stop = "none"
+        a_p_e.doevtgrade = False
+        a_p_e.apply_cti = True
+        a_p_e.apply_tgain = True #False
+        a_p_e.calculate_pi = True
+        a_p_e.pix_adj = "NONE"
+        a_p_e.eventdef = eventdef
+        a_p_e.clobber = True
+        a_p_e.verbose = verbose
+
+        a_p_e.gainfile = f"{aciscaldb}/det_gain/{gainfile}"
+        a_p_e.ctifile = f"{aciscaldb}/t_gain/{tgainfile}"
+        a_p_e.tgainfile = f"{aciscaldb}/cti/{ctifile}"
 
         try:
-            acis_process_events.infile = infile
-            acis_process_events()
+            a_p_e.infile = infile
+            a_p_e()
```

Comment:
you should just be able to say

    a_p_e(infile=infile)

here but again not an important change.